### PR TITLE
Nice Documentation

### PR DESCRIPTION
**Pull request by davidsiaw**
This PR adds some nice documentation!

View it at https://tech.degica.com/kaiser/branches/nice-docs/0110-getting_started/

### CODE REVIEW

**Comment by degicat** [comment]
@pmq20 can you help us review this PR, please?

File: lib/kaiser/plugin.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@ def self.loaded?(name)
     def self.inherited(plugin)
       super
 
+      puts "INHERITED #{plugin}"
```
Comment (pmq20) [inline lib/kaiser/plugin.rb:38]: Is this for temporary debugging? Was this line intended to be committed to the git repo?

**Review by pmq20 — APPROVED** [review]
Thanks for adding documentation! I tried building the docs and it worked.

Just a small question. Otherwise LGTM.

image

File: lib/kaiser/plugin.rb (hunk 1 above)
Comment (davidsiaw) [reply lib/kaiser/plugin.rb:38]: o yes. will remove

**Pull request merged**